stoplight.js

```diff
@@ -0,0 +1,48 @@
+$("document").ready(function(){
+  console.log("hello");
+
+
+  var $stopLight = $("#stopLight")
+  var redlight = $("#red");
+  var yellowlight = $("#yellow");
+  var greenlight = $("#green");
+  var color;
+  var working;
+
+
+  function changeColor(bulb,working){
+    working = !true
+    if (working == !true ){
+      bulb.css("background-color", color);
+    }
+    if(working == true){
+      bulb.css("background-color", "grey");
+    }
+  };
```

Comment:
I liked that you intuited this is a place where you could use a flag! 

I think it might be simpler without it, though. Whether or not an element has a class of `active` can act like a sort of built-in toggle/flag.